src/components/3d-hover-effect-glow/3d-hover-effect-glow.tsx

```diff
@@ -0,0 +1,24 @@
+import { Component, h, Host } from '@stencil/core';
+/**
+ * This component enhances the visual effects, when the `tiltFollowingTheCursor`
+ * utility function from `3d-tilt-hover-effect.ts` is implemented in a component.
+ *
+ * This component should be added to the HTML structure of the consumer component.
+ *
+ * Styles of this component are needed to create a glow effect on the
+ * 3D element within the when the parent element is hovered.
+ * when the `tiltFollowingTheCursor` utility function from `3d-tilt-hover-effect.ts`
```

Comment:
This line seems to have been left from an earlier iteration? Or something else is wrong. It doesn't quite fit in, at any rate 🙂

Reply:
perhaps this could be better formulated. I was trying to say if you want the glow effect, you're gonna need this component (and the styles that it carries) kind of 😓

Reply:
Ah, maybe all that's wrong with it is that the period is at the end of the second line instead of at the end of the third?

Reply:
Feel free to update it as you wish bro. My head is everywhere now, and I am trying to catch up with a deadline. 🙈